ui/app/AppLayouts/Wallet/views/collectibles/CollectibleDetailView.qml

```diff
@@ -20,6 +22,11 @@ Item {
     property bool isCollectibleLoading
     readonly property int isNarrowMode : width < 700
 
+    // Community related token props:
+    property int privilegesLevel: Constants.TokenPrivilegesLevel.Community // TO BE REMOVED. It should be just in `collectible.privilegesLevel`
+    readonly property bool isOwnerTokenType: privilegesLevel === Constants.TokenPrivilegesLevel.Owner//!!collectible.privilegesLevel ? (collectible.privilegesLevel === Constants.TokenPrivilegesLevel.Owner) : false
+    readonly property bool isTMasterTokenType: privilegesLevel === Constants.TokenPrivilegesLevel.TMaster//!!collectible.privilegesLevel ? (collectible.privilegesLevel === Constants.TokenPrivilegesLevel.TMaster) : false
+
```

Comment:
Commented out code? Tracked in some other issue, or will be fixed/ammended in the parent PR?

Reply:
It's been ammended in here!